src/snakemake/api.py

```diff
@@ -640,13 +641,15 @@ def create_report(
         self,
         reporter: str = "html",
         report_settings: Optional[ReportSettingsBase] = None,
+        global_report_settings: Optional[GlobalReportSettings] = None,
     ):
         """Create a report for the workflow.
 
         Arguments
         ---------
         report: Path -- The path to the report.
-        report_stylesheet: Optional[Path] -- The path to the report stylesheet.
```

Comment:
Thinking about it, maybe we should keep the stylesheet, even if it is currently ignored. And then, in a separate work, restore its functionality. This way, people can restyle reports as needed for their institution (e.g. change the color scheme).

Reply:
Ah, understood. In that case it is fine like this.